### ISSUE TITLE
Build fails.

### ISSUE THREAD

**Issue by atomerz** [castlabs/dashencrypt#22]
When I try to build the source it gives me this error:
> Failed to execute goal on project dash.fragmencrypter: Could not resolve dependencies for project 
> com.castlabs.dash:dash.fragmencrypter:jar:1.7-SNAPSHOT: Failure to find 
> com.googlecode.mp4parser:isoparser:jar:1.0.7.2-SNAPSHOT in 
> https://oss.sonatype.org/content/repositories/comgooglecodemp4parser-1062 was cached in the local 
> repository, resolution will not be reattempted until the update interval of mp4parserstaging has
> elapsed or updates are forced

**Comment by sannies** [comment]
A quick fix would be to build the isoparser project locally on your
maschine (https://github.com/sannies/mp4parser). I'll build a isoparser
release at some point next week.

2015-06-25 21:16 GMT+02:00 atomerz <notifications@github.com>:

**Comment by atomerz** [comment]
Thanks, I managed to build the app and DASH some content with it. But the problem is that the output cannot be played in my browser (Chrome Version 43.0.2357.130 m) using dash.js player. I think it because of the audio codec (mp4a.40.5) of the produced output. Even ffmpeg does recognize it. When I compared it to other samples over the internet I noticed their codec is mp4a.40.2.  
Is this a problem with dashencrypt or am I doing something wrong?
